### DISCUSSION TITLE
My student id is approved but I didn't get my benefits

### DISCUSSION THREAD

**Discussion by MuskanMandal15** [community/community#138785] [GitHub Education]
### Hi everyone,

When would I receive my welcome benefits for github student developer pack

**Comment by ezwelt** [comment]
❤️

**Comment by talha1230** [comment]
got to this link and check whether its approved or not:
[GitHub Education benefits](https://education.github.com/discount_requests/application)
if not try submitting application again with verified documents; you should see this is approved
![image](https://github.com/user-attachments/assets/0b1f2bbe-dbef-431e-a237-a16323f96361)

**Reply by MuskanMandal15** [reply]
This is what I am saying my request is approved but I did not get any benefits yet, and almost 3 days have passed

**Reply by doxybox** [reply]
I have that problem self!

**Reply by MuskanMandal15** [reply]
Hi @davevad93, can you look into the matter

**Reply by davevad93** [reply]
Hey @MuskanMandal15, I'm not affiliated with GitHub or the Education team so I can't help you expedite your process, however I can give you some helpful information.

First of all, what's the status of your application?

If the label is green and states "`Congrats! Your request for @MuskanMandal15 was approved and your benefits will be available soon.` that means you have to wait. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; the waiting time may vary depending on different factors, such as the volume of requests.

If the label is purple and states "`Congrats! Your request for @MuskanMandal15 was approved and your benefits are now available.`", that means you should have access to your benefits. Try first going to the [pack offers page](https://education.github.com/pack) and select one of the available offers, if you can access them you are fine, if not I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue.

For further reference, take a look at the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352):

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

And also the following discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721

**Comment by MuskanMandal15** [comment]
Resolved
